Boop/Boop/scripts/Rot13.js

```diff
@@ -0,0 +1,21 @@
+/**
+  {
+    "api":1,
+    "name":"Rot13",
+    "description":"Applies the Rot13 cypher to your text",
+    "author":"Paul Starr",
+    "icon":"flip",
+    "tags":"spoilers,encryption,plaintext"
+  }
+**/
+
+function main(state) {
+	let myText = state.text
+	// adapted from Sophie Alpert's solution: https://stackoverflow.com/questions/617647/where-is-my-implementation-of-rot13-in-javascript-going-wrong
+	state.fullText = myText.replace(/[a-z]/gi, function (c) {
```

Comment:
You're replacing the full text (`state.fullText`) with selected text (`state.text`) . Is this intentional?